### PR TITLE
Dialogs UI

### PR DESCRIPTION
**Pull request by lucianoratamero**
**Link to task:** https://labcodes.atlassian.net/browse/DSYS-23
**Staging app:** https://dialogs-ui--labstorybook-master.netlify.app/?path=/docs/components-dialog--page

This PR implements the UI on Dialogs.
We tried fixing the git tree so that it wouldn't take this many commits, but something went wrong upstream and we didn't want to waste more time on this. Everything was properly validated, and changes are synced with master.

@debscorreia , please try the mobile versions using the mobile emulator in devtools as well. :]

### CODE REVIEW

File: labsystem/scss/components/_lab-dialog.scss [hunk 1]
```diff
@@ -1,85 +1,407 @@
...
+  border: none;
+
+  &.lab-dialog__close-button--message {
+    display: flex;
+    align-items: center;
+  }
+
+  &:focus {
+    border-radius: $radius-02;
+    box-shadow: 0 0 0 2px $teal-30;
```
Comment (debscorreia) [inline labsystem/scss/components/_lab-dialog.scss:213]: I noticed we are defining colors directly and not using from the theme. Shouldn't dialogs follow the chosen theme?

Comment (debscorreia) [reply labsystem/scss/components/_lab-dialog.scss:213]: Also, just remembered we have a mixin for the focus shadow that we could use. On mixins.scss, line 6.

**Review by debscorreia — COMMENTED** [review]
Hey @lucianoratamero @CavalcanteLucas I left some comments. It was a little tricky to review this PR due to the amount of things, but I focused on the lab-dialogs.scss file and left some comments on jira. 

The only "big" thing I noticed here on the code is that we are not using a theme for the dialogs. Was that a decision made? 

Other than that, just small things :)

File: playgrounds/DialogPlayground.js [hunk 2]
```diff
@@ -214,19 +215,31 @@ export default class DialogPlayground extends React.Component {
           {!windowIsSmall ? (
             <>
```
Comment (CavalcanteLucas) [inline playgrounds/DialogPlayground.js:216]:
```suggestion
            <React.Fragment>
```

File: playgrounds/DialogPlayground.js [hunk 3]
```diff
@@ -214,19 +215,31 @@ export default class DialogPlayground extends React.Component {
...
-                  checked={selectedIsModal}
-                  label="isModal"
-                  disabled={modalIsOpen}
+                  label="selectedIsModal"
+                  value={selectedIsModal}
+                  handleToggle={this.handleBoolPropChange}
                 />
               </span>
-              <br />
             </>
```
Comment (CavalcanteLucas) [inline playgrounds/DialogPlayground.js:228]:
```suggestion
            </React.Fragment>
```

**Review by CavalcanteLucas — APPROVED** [review]

**Pull request merged**